fusor-ember-cli/app/routes/review/progress/overview.js

```diff
@@ -27,10 +27,23 @@ export default Ember.Route.extend({
       });
   },
 
+  setupController: function(controller, model) {
+    controller.set('model', model);
+    //alert("startPolling in setupController");
+    controller.stopPolling();
+    controller.startPolling();
+  },
+
+  deactivate: function() {
+    //alert("stopPolling in deactivate");
+    this.get('controller').stopPolling();
+  },
+
   actions: {
-    refreshModel: function(){
-      console.log('refreshModelOnRoute');
+    refreshModelOnOverviewRoute: function(){
+      console.log('refreshModelOnOverviewRoute');
       return this.refresh();
```

Comment:
The original code with ONE progress bar used `this.get('model').reload()` which worked when the model returned ONE `foreman-task`. However, it didn't work now that `model` returns
```
        return Ember.RSVP.hash({
           deployTask: deployTask,
           manageContentTask: manageContentTask,
           rhevTask: rhevTask,
           cfmeTask: cfmeTask
        });
```
So, I'm using `this.refresh()` which refreshes the whole route including `model` and `setupController`. This is why, I needed to call `stopPolling` in setupController. Otherwise, it would start a bunch of loops.